resources/ec2_instances.go

```diff
@@ -351,7 +352,7 @@ func Ec2Instances() *schema.Table {
 				Resolver:    fetchEc2InstanceBlockDeviceMappings,
 				Columns: []schema.Column{
 					{
-						Name:        "instance_id",
+						Name:        "instance_cq_id",
```

Comment:
Should we add instance_id field that points to the instance id primary key of the parent?

Reply:
yes, we can also do that with ParentFieldResolver